app/configurations/config.default.js

```diff
@@ -214,9 +216,6 @@ export default {
   // if you enable car suggestions but the linear distance between all points is less than this, then a car route will
   // not be computed
   suggestCarMinDistance: 2000,
-  minTransferTime: 90,
-  optimize: 'GREENWAYS',
-  transferPenalty: 0,
```

Comment:
Was this unused?

Reply:
No, but those 3 settings are now moved to defaultSettings block so that they do not require special handling but work  the same way as all other routing settings.

Reply:
I had just missed that the transferPenalty was already defined in the defaultSettings, that's why it didn't need to be added there.